pulsar-sql/presto-pulsar/src/test/java/org/apache/pulsar/sql/presto/decoder/protobufnative/TestMsg.java

```diff
@@ -190,7 +190,7 @@ private SubMessage(
         throws com.google.protobuf.InvalidProtocolBufferException {
       this();
       if (extensionRegistry == null) {
-        throw new java.lang.NullPointerException();
+        throw new java.lang.IllegalArgumentException();
```

Comment:
Looks like the TestMsg is generated by protobuf, we can skip this class.

Reply:
👍🏼

Reply:
@codelipenghui I've reverted the changes inside `TestMsg` file